preview-src/ui-model.yml

```diff
@@ -1,127 +1,129 @@
   next:
     content: Liber Recusabo
-    url: '#'
-    urlType: 'internal'
+    url: "#"
+    urlType: "internal"
   breadcrumbs:
-  - content: Quickstart
-    url: '#'
-    urlType: fragment
-  - content: Brand&#8217;s Hardware &amp; Software Requirements
-    url: /bonita/7.10/index.html
-    urlType: internal
-  versions:
-  - version: '7.11'
-    displayVersion: '7.11'
-    url: '#'
-  - version: '7.10'
-    displayVersion: '7.10'
-    url: '#'
-  - version: '7.9'
-    displayVersion: '7.9'
-    url: '#'
-  - version: '7.8'
-    displayVersion: '7.8'
-    missing: true
-    url: '#'
-  navigation:
-  - root: true
-    content: Project Bonita
-    url: '#'
-    urlType: fragment
-    items:
     - content: Quickstart
-      url: '#'
+      url: "#"
       urlType: fragment
-      items:
-      - content: Brand&#8217;s Hardware &amp; Software Requirements
-        url: /bonita/7.10/index.html
-        urlType: internal
-      - content: Overview
-        url: '/bonita/7.10/overview.html'
-        urlType: internal
-      - content: Misc
-        url: '/bonita/7.10/misc.html'
-        urlType: internal
-    - content: Liber Recusabo
-      url: '#liber-recusabo'
+    - content: Brand&#8217;s Hardware &amp; Software Requirements
+      url: /bonita/7.10/index.html
+      urlType: internal
+  versions:
+    - version: "7.11"
+      displayVersion: "7.11"
+      url: "#"
+    - version: "7.10"
+      displayVersion: "7.10"
+      url: "#"
+    - version: "7.9"
+      displayVersion: "7.9"
+      url: "#"
+    - version: "7.8"
+      displayVersion: "7.8"
+      missing: true
+      url: "#"
+  navigation:
+    - root: true
+      content: Project Bonita
+      url: "#"
       urlType: fragment
-    - content: Reference
       items:
-      - content: Keyboard Shortcuts
-        url: '#'
-        urlType: fragment
-      - content: Importing and Exporting
-        url: '#'
-        urlType: fragment
-      - content: Some Code
-        url: '/bonita/7.10/index.html#some-code'
-        urlType: internal
+        - content: Quickstart
+          url: "#"
+          urlType: fragment
+          items:
+            - content: Brand&#8217;s Hardware &amp; Software Requirements
+              url: /bonita/7.10/index.html
+              urlType: internal
+            - content: Overview
+              url: "/bonita/7.10/overview.html"
+              urlType: internal
+            - content: Misc
+              url: "/bonita/7.10/misc.html"
+              urlType: internal
+        - content: Liber Recusabo
+          url: "#liber-recusabo"
+          urlType: fragment
+        - content: Reference
+          items:
+            - content: Keyboard Shortcuts
+              url: "#"
+              urlType: fragment
+            - content: Importing and Exporting
+              url: "#"
+              urlType: fragment
+            - content: Some Code
+              url: "/bonita/7.10/index.html#some-code"
+              urlType: internal
```

Comment:
Please stick with the convention:
- Do not double the indentation
- Do not use double quote instead of simple quote

Reply:
I am on it